src/components/Content.jsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from "react";
+import Card from "./Card";
+
+const Content = ({ setPage }) => {
+	useEffect(() => {
+		setPage("home");
+	}, []);
```

Comment:
React is giving you a warning when you npm run start, this warning should be fixed